README.md

```diff
@@ -1,44 +1,53 @@
 # JustBar
+A HMOS library to add a Bar.
 
-![](https://github.com/Hamadakram/JustBar/blob/master/art/banner.gif)
-## Download
-Grab via Gradle:
-```groove
-implementation 'com.irozon.justbar:justbar:1.0.1'
+# Source
+Inspired by [Hammad Akram/JustBar](https://github.com/Hamadakram/JustBar) - version 1.0
+
+## Features
+This library provides an animation similar to Just bar feature.
```

Comment:
Features section must be more descriptive of the features in the lib. Please check this.

Reply:
Done